__tests__/app.test.js

```diff
@@ -128,6 +128,46 @@ beforeEach(() => {
                 })
             })
         })
+
+        describe('GET /api/articles/:article_id/comments', () => {
+            test('Returns an array of comments for the requested article, comments should be returned with the most recent first. Each article should have a comment_id, votes, created_at, author, body, article_id', () => {
+                return request(app)
+                .get('/api/articles/1/comments')
+                .expect(200)
+                .then((res) => {   
+                    expect(Array.isArray(res.body.comments)).toBe(true);
+                    res.body.comments.forEach((comment) => {
+                    expect(comment).toEqual(expect.objectContaining({
+                       comment_id: expect.any(Number), 
+                       votes: expect.any(Number),
+                       created_at: expect.any(String),
+                       author: expect.any(String),
+                       body: expect.any(String),
+                       article_id: expect.any(Number),
+                       
+                    })
+                    )
+                })
+
+                expect(res.body.comments.length).toBe(11);
```

Comment:
Very minor thing for me to pick up on, but testing for the length might be better done before iterating through the comments.